tests/e2e/utils/measure.ts

```diff
@@ -0,0 +1,36 @@
+import {profiler} from '@perf-profiler/profiler';
+import {getAverageCpuUsage, getAverageFPSUsage, getAverageRAMUsage} from '@perf-profiler/reporter';
+import type {Measure} from '@perf-profiler/types';
+import noop from 'lodash/noop';
+
+let measures: Measure[] = [];
+let polling = {
+    stop: noop,
```

Comment:
NIT: maybe not `noop`, but a function that throws an error, so that we are explicit?